app/src/main/java/com/yello/presentation/main/recommend/RecommendInviteDialog.kt

```diff
@@ -0,0 +1,84 @@
+package com.yello.presentation.main.recommend
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import com.example.ui.base.BindingDialogFragment
+import com.example.ui.fragment.snackBar
+import com.example.ui.fragment.toast
+import com.example.ui.view.setOnSingleClickListener
+import com.kakao.sdk.common.util.KakaoCustomTabsClient
+import com.kakao.sdk.share.ShareClient
+import com.kakao.sdk.share.WebSharerClient
+import com.yello.R
+import com.yello.databinding.FragmentRecommendInviteDialogBinding
+import timber.log.Timber
+
+class RecommendInviteDialog :
+    BindingDialogFragment<FragmentRecommendInviteDialogBinding>(R.layout.fragment_recommend_invite_dialog) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.btnInviteDialogExit.setOnSingleClickListener {
+            dismiss()
+        }
+
+        binding.btnInviteKakao.setOnSingleClickListener {
+            startKakaoInvite(this.requireContext())
+        }
+
+        binding.btnInviteLink.setOnSingleClickListener {
+            startLinkInvite()
+        }
+    }
+
+    // TODO : 스낵바 커스텀 & 링크 복사
+    private fun startLinkInvite() {
+        requireParentFragment().snackBar(requireParentFragment().requireView()) { "링크가 복사되었습니다." }
+        requireParentFragment().toast("링크가 복사되었습니다.")
+    }
+
+    // TODO : 기획에서 제공해줄 템플릿, 링크에 맞춰서 추후 수정
+    private fun startKakaoInvite(context: Context) {
+
+        // 사용자 정의 템플릿 ID & 공유할 url
+        val templateId = 95890.toLong()
+        val url = "http://locahost"
```

Comment:
```suggestion
        private val templateId = 95890.toLong()
        private val url = "http://locahost"
```
외부에서 사용되지 않는 변수나 함수는 private으로 지정하는 것이 효율적일 것 같습니다!